src/App.js

```diff
@@ -1,25 +1,15 @@
-import './App.css';
+import './App.scss';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Everything in your project is perfect and follows the requirement except for only a small issue. Kindly change the `App` component to be `class-based` instead of `functional-based` as it's the requirement for this project.